### PR TITLE
Fix  camelize arguments for subscriptions event IDs

### PR DESCRIPTION
**Pull request by durcak**

### CODE REVIEW

**Comment by durcak** [comment]
TO fix issue [https://github.com/rmosolgo/graphql-ruby/issues/2553](url)

File: lib/graphql/subscriptions/event.rb [hunk 1]
```diff
@@ -37,7 +37,7 @@ def self.serialize(name, arguments, field, scope:)
           arguments
         when Hash
           if field.is_a?(GraphQL::Schema::Field)
-            stringify_args(arguments)
+            stringify_args(arguments, field)
```
Comment (rmosolgo) [inline lib/graphql/subscriptions/event.rb]: The definition below has `arg_owner, args` for the parameters, should these be switched?

```suggestion
            stringify_args(field, arguments)
```

Comment (durcak) [reply lib/graphql/subscriptions/event.rb]: Ou, yes it should.

Comment (durcak) [reply lib/graphql/subscriptions/event.rb]: So, I have updated the tests and add small fix for stringify_args method with auto-loaded arguments, I am not sure, if it is the best solution, so I am open for a next discusion and suggestions.

**Comment by durcak** [comment]
So, I have updated the tests and add small fix for auto-loaded arguments, I am not sure, if it is the best solution, so I am open for a next discusion or suggestions.

**Comment by rmosolgo** [comment]
It looks good to me! I don't know of a better way of finding a match. Thanks for taking care of this!

**Pull request merged**